packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/channels/NotificationsChannelsProvider.kt

```diff
@@ -0,0 +1,13 @@
+package expo.modules.notifications.notifications.channels
+
+import expo.modules.notifications.notifications.channels.managers.NotificationsChannelGroupManager
+import expo.modules.notifications.notifications.channels.managers.NotificationsChannelManager
+import expo.modules.notifications.notifications.channels.serializers.NotificationsChannelGroupSerializer
+import expo.modules.notifications.notifications.channels.serializers.NotificationsChannelSerializer
+
+interface NotificationsChannelsProvider {
```

Comment:
I had thought that all interfaces went into the "interfaces" folder?

Reply:
In this case the kotlinified file has the same location as the original java one.
I'll move it in some other PR if it fits better.

Reply:
Ah, then it's ok as is, no need to move it 👍